sdk/avalon_sdk/fabric/fabric_work_order.py

```diff
@@ -104,9 +104,35 @@ def work_order_get_result(self, work_order_id, id=None):
         # take some time to commit to chain.
         # Instead of calling contract api to get result chosen the
         # event based approach.
+        def handle_fabric_event(event, block_num, txn_id, status):
```

Comment:
In a later PR , we can use a common approach for event handling with asyncio construct for both fabric and ethereum